install/kubernetes/Makefile

```diff
@@ -40,3 +44,28 @@ docs:
 .PHONY: $(TETRAGON_CHART)/crds-yaml
 $(TETRAGON_CHART)/crds-yaml: $(CRDS)
 	cp -rf $(CRDS)/. $(TETRAGON_CHART)/crds-yaml
+
+# openapi2jsonschema.py script generating JSON schema from the CRD YAML spec.
+.PHONY: openapi2jsonschema.py
+openapi2jsonschema.py:
+	curl -sSfLO https://raw.githubusercontent.com/yannh/kubeconform/$(KUBECONFORM_VERSION)/scripts/$@
+
+# To validate (using openapi2jsonschema.py) default Ruleset policies included in the Helm chart, we need to pass the
+# JSON schema of the TracingPolicy CRD. This target generates such schema.
+# Requires pipenv to be pre-installed.
+.PHONY: generate-jsonschemas
+generate-jsonschemas: $(CRDS)
```

Comment:
Background information: I implemented this because the `ubuntu-latest` GH runner has `pyyaml` installed, but many OS (like macOS) don't. Also, it's not in Python's standard library. Hence, it would fail locally on many OSes. Furthermore, `pipenv` isn't installed on the `ubuntu-latest` GH runner, so if we would like to go with the pipenv route everywhere, we would need to install it in the GH actions.

I'm also fine with doing everything using `pipenv`.